test/modeling_library/distributions.jl

```diff
@@ -172,8 +172,8 @@ end
     @test isapprox(actual[1][2], finite_diff_vec(f, args, 1, 2, dx))
     @test isapprox(actual[2][1], finite_diff_vec(f, args, 2, 1, dx))
     @test isapprox(actual[2][2], finite_diff_vec(f, args, 2, 2, dx))
-
-    @test actual[3] === nothing # not yet implemented
+    @test isapprox(actual[3][1, 1], finite_diff_mat_sym(f, args, 3, 1, 1, dx))
+    @test isapprox(actual[3][1, 2], finite_diff_mat_sym(f, args, 3, 1, 2, dx))
```

Comment:
Seems like it would make sense to check the other 2 entries in the matrix too?

Reply:
Thanks a lot Sam -- this is great. I just had one question about the test.

Reply:
Good catch. Just added below.